generated/nidaqmx/_library_interpreter.py

```diff
@@ -13,6 +13,15 @@ class LibraryInterpreter(BaseInterpreter):
     This class is responsible for interpreting the Library's C API.
 
     """
+    
+    def __init__(self):
+        # These lists keep C callback objects in memory as ctypes doesn't.
+        # Program will crash if callback is made after object is garbage
+        # collected.
+        self._done_event_callbacks = []
+        self._every_n_samples_event_callbacks = []
```

Comment:
Previously, this had separate lists for `EveryNSamplesEventType.TRANSFERRED_FROM_BUFFER` and `EveryNSamplesEventType.ACQUIRED_INTO_BUFFER`. If DAQmx supports using both event types in the same task, then combining their lists introduces a regression.

@zhindes Do you know if any devices support this?